Sources/CreateAPI/Generator/Generator+Render.swift

```diff
@@ -81,7 +81,12 @@ extension Generator {
                 var needsValues = false
                 let decoderContents = properties.map {
                     if case .userDefined = $0.type {
-                        return templates.decodeFromDecoder(property: $0)
+                        if $0.isInlined ?? false {
+                            needsValues = true
+                            return templates.decode(property: $0, isUsingCodingKeys: false)
```

Comment:
I think we can avoid the nested if (and the duplication on the lines below) by flipping this check and including it on the outer if statement.

Would this produce the same result?

```swift
if case .userDefined = $0.type, $0.isInlined != true {
    return templates.decodeFromDecoder(property: $0)
} else {
    needsValues = true
    return templates.decode(property: $0, isUsingCodingKeys: false)
}
```

Reply:
Thanks! Fixed in [b276bb8](https://github.com/CreateAPI/CreateAPI/pull/27/commits/b276bb8d8c2b3204da8b43ac1890dcf66bab7259).